drivers/bluetooth/hci/rpmsg.c

```diff
@@ -317,9 +325,55 @@ static int bt_rpmsg_open(void)
 	return 0;
 }
 
+static int bt_rpmsg_close(void)
+{
+	int err;
+	struct net_buf *buf = NULL;
+	struct net_buf *rsp = NULL;
```

Comment:
Unnecessary initialisation upon declaration (just remove the `= NULL`

Reply:
Removed.